hadoop-tools/hadoop-aws/src/main/java/org/apache/hadoop/fs/s3a/tools/MarkerTool.java

```diff
@@ -960,43 +960,64 @@ public static final class ScanArgsBuilder {
     /** Consider only markers in nonauth paths as errors. */
     private boolean nonAuth = false;
 
-    /** Source FS; must be or wrap an S3A FS. */
+    /** Source FS; must be or wrap an S3A FS.
```

Comment:
the comment should start in newline not in the **. The same goes for all the comments below at multiple places.

Reply:
Do you think it's a good idea adding this to CheckStyle?

This one looks relevant (though maybe it will flag a lot of existing files): https://checkstyle.sourceforge.io/config_javadoc.html#JavadocContentLocation

Reply:
Started all the comments from new line MarkerTool.java. Thanks Mukund and Danny for the feedback.

Reply:
Let's not change the check style as yes it will flag many issues.

Reply:
>Let's not change the check style as yes it will flag many issues.

It will flag issues, but maybe worth it to avoid needing to add this sort of feedback in future.
Let Yetus tell us before we review.

If were were to consider it, I'd propose to do it:
- In a separate PR / task so we can acknowledge "yes this is adding new warnings".
- Only for `hadoop-aws` module.

Reply:
Sure we can try only for hadoop-aws module.